bellman/src/lib.rs

```diff
@@ -26,6 +26,10 @@ mod multiexp;
 #[cfg(test)]
 mod tests;
 
+pub fn get_chunk_size(total: usize) -> usize {
+    (total - 1) / num_cpus::get() + 1
```

Comment:
This will panic if total is equal to zero.

Reply:
Fixed in [new commit](https://github.com/MystenLabs/phase2-bn254/pull/1/commits/2fcbba8856664f80902cbf06d3a80b548dd9312e#diff-712193d9f325ef870c241172009a26cfe5bed8ace06fc0c81cf660e9bbe654bcR30-R34).